neo/Network/P2P/TaskManager.cs

```diff
@@ -224,8 +224,8 @@ private void RequestTasks(TaskSession session)
             }
             if ((!HasHeaderTask || globalTasks[HeaderTaskHash] < MaxConncurrentTasks)
                 && (Blockchain.Singleton.HeaderHeight < session.StartHeight
-                    || (Blockchain.Singleton.HeaderHeight == session.StartHeight
-                        && Blockchain.Singleton.HeaderHeight == sessions.Select(x => x.Value.StartHeight).Max()
+                    || (Blockchain.Singleton.Height == Blockchain.Singleton.HeaderHeight
+                        && Blockchain.Singleton.HeaderHeight >= sessions.Select(x => x.Value.StartHeight).Max()
```

Comment:
@yongjiema, in my opinion it could even be the oposite:

`Blockchain.Singleton.HeaderHeight < sessions.Select(x => x.Value.StartHeight).Max()`

Request when we known that some session has a `StartHeight` higher than we are.

Anyway...let's keep it like this...aehuaheua